src/modules/dashboard/ui/components/dashboard-user-button.tsx

```
@@ -0,0 +1,78 @@
+import { useRouter } from "next/navigation";
+import { ChevronDownIcon, CreditCardIcon, LogOutIcon } from "lucide-react";
+
+import { Avatar, AvatarImage } from "@/components/ui/avatar";
+import { GeneratedAvatar } from "@/components/generated-avatar";
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuItem,
+    DropdownMenuLabel,
+    DropdownMenuSeparator,
```

ESLint

[error] 'DropdownMenuSeparator' is defined but never used.

(@typescript-eslint/no-unused-vars)